Example/CardScan/ViewController.swift

```diff
@@ -24,10 +23,29 @@ class ViewController: UIViewController, ScanEvents, ScanDelegate, FullScanString
     
     var currentTestImages: [CGImage]?
     
-    func nextImage() -> CGImage? {
-        let nextImage = self.currentTestImages?.first
+    func nextSquareAndFullImage() -> (CGImage, CGImage)? {
+        guard let fullCardImage = self.currentTestImages?.first else {
+            print("could not get full size test image")
+            return nil
+        }
+        
+        let cropToSquareImage: CGImage? = {
```

Comment:
why are you doing this with a closure?

Reply:
i just did it so i could encapsulate it. no strong feelings about it. should i change it?